### DISCUSSION TITLE
Image Download FVD Ep 2

### DISCUSSION THREAD

**Discussion by codetricity** [caitlynbakery/flutter_video_display_8#2] [Ideas]
path provider is from Flutter and is a good place to start.

https://pub.dev/packages/path_provider - 2921 likes, 100% popularity, Flutter Favorite

I think there are numerous packages to get the path on Android.

https://pub.dev/packages/path_provider_android
https://pub.dev/packages/android_path_provider

## Goal

Save the image to a location that the Flutter app can access.

![image](https://user-images.githubusercontent.com/5799860/178500428-a8d2eb6c-26a6-4f4a-8835-0125125429ab.png)

You likely need an image picker
https://pub.dev/packages/image_pickers

https://pub.dev/packages/image_picker

I have never been able to implement this effectively.

**Comment by codetricity** [comment]
# Discussion 7/12/2022

![image](https://user-images.githubusercontent.com/5799860/178516232-af018c41-0784-49aa-a782-628e06b758a9.png)

* use phone gallery app to test if saved

__optional__

* implement image picker to see image

**Reply by caitlynbakery** [reply]
Ok, I will implement the main objective of the app: saving the image to local storage.
